lib/usdUfe/ufe/UsdContextOps.cpp

```diff
@@ -508,4 +713,37 @@ UsdContextOps::SchemaNameMap UsdContextOps::getSchemaPluginNiceNames() const
     return schemaPluginNiceNames;
 }
 
+static_assert(std::has_virtual_destructor<Ufe::CompositeUndoableCommand>::value);
+static_assert(std::is_base_of<
+              UsdBulkEditCompositeUndoableCommand::Parent,
+              UsdBulkEditCompositeUndoableCommand>::value);
+
+void UsdBulkEditCompositeUndoableCommand::execute()
+{
+    // Same as base class in forward order.
+    // Iterate on our internal command list and only add to base class any commands
+    // which succeed (no error thrown). Thus we do not need to override undo/redo.
+    for (const auto& cmd : _cmds) {
+        if (cmd) {
+            try {
+                cmd->execute();
+                Parent::append(cmd);
```

Comment:
Try each command and only keep the ones that succeed. Some might fail because of edit restrictions or not being applicable to the item.